### PR TITLE
Fix: UI and text adjustments

### PR DESCRIPTION
**Pull request by tewshi**
## Checklist

- [x] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/locales/gr.json [hunk 1]
```diff
@@ -567,8 +567,8 @@
         "title": "History Events Query"
       },
       "missing_rule": {
-        "title": "Το συμβάν δεν υποβλήθηκε σε επεξεργασία",
-        "message": "Για να το χειριστείτε, πρέπει να επιλέξετε μία από αυτές τις επιλογές:",
+        "title": "Λογιστικός κανόνας για το γεγονός που λείπει",
+        "message": "Αυτό το συμβάν δεν θα υποβληθεί σε επεξεργασία στη λογιστική. Για να το χειριστείτε, πρέπει να επιλέξετε μία από αυτές τις επιλογές:",
```
Comment (kelsos) [inline frontend/app/src/locales/gr.json:571]:
```suggestion
        "message": "Το γεγονός αυτό δεν θα υποβληθεί σε λογιστική επεξεργασία. Για να το χειριστείτε, πρέπει να επιλέξετε μία από αυτές τις επιλογές:",
```

File: frontend/app/src/locales/gr.json [hunk 2]
```diff
@@ -2044,7 +2044,7 @@
     },
     "blockscout": {
       "delete_tooltip": "Διαγράφει το κλειδί Blockscout API",
-      "description": "Το rotki χρησιμοποιεί το Blockscout για να λάβει πληροφορίες σχετικά με τα ερωτήματα ανάληψης ETH",
+      "description": "Το rotki χρησιμοποιεί το Blockcout για να λάβει πληροφορίες σχετικά με ερωτήματα ανάληψης ETH και πολλά άλλα. Ένα κλειδί API απαιτείται μόνο εάν σας περιορίζουν τα ποσοστά.",
```
Comment (kelsos) [inline frontend/app/src/locales/gr.json:2047]:
```suggestion
      "description": "Το rotki χρησιμοποιεί το Blockscout για να λαμβάνει πληροφορίες σχετικά με τα ερωτήματα ανάληψης ETH και άλλα. Ένα κλειδί API είναι απαραίτητο μόνο αν έχετε περιορισμένο ρυθμό.",
```

File: frontend/app/src/locales/es.json [hunk 3]
```diff
@@ -571,8 +571,8 @@
         "title": "History Events Query"
       },
       "missing_rule": {
-        "title": "Evento no procesado",
-        "message": "Para manejarlo, debes elegir una de estas opciones:",
+        "title": "Evento que falta regla contable",
```
Comment (kelsos) [inline frontend/app/src/locales/es.json:574]: @yabirgb can you check the changes here?

**Review by kelsos — APPROVED** [review]
I am leaving only some suggestions for the translation here.

The changes and the code look good thanks @tewshi

**Review by kelsos — CHANGES_REQUESTED** [review]
I realized that we can accidentally merge this way so I am gonna block until @yabirgb reviews the Spanish translation

File: frontend/app/src/locales/es.json (hunk 3 above)
Comment (yabirgb) [inline frontend/app/src/locales/es.json:574]:
```suggestion
        "title": "Evento al que le falta una regla contable",
```

File: frontend/app/src/locales/es.json [hunk 4]
```diff
@@ -571,8 +571,8 @@
...
+        "message": "Este evento no será procesado en contabilidad. Para manejarlo, debes elegir una de estas opciones:",
```
Comment (yabirgb) [inline frontend/app/src/locales/es.json:575]:
```suggestion
        "message": "Este evento no será procesado en el reporte de ganancias. Para corregirlo, debes elegir una de estas opciones:",
```

File: frontend/app/src/locales/es.json [hunk 5]
```diff
@@ -2038,7 +2038,7 @@
     },
     "blockscout": {
       "delete_tooltip": "Elimina la clave API de Blockscout",
-      "description": "rotki utiliza Blockscout para obtener información sobre las consultas de retirada de ETH",
+      "description": "rotki usa Blockscout para obtener información sobre consultas de retiro de ETH y más. Solo se necesita una clave API si tiene una tarifa limitada.",
```
Comment (yabirgb) [inline frontend/app/src/locales/es.json:2041]:
```suggestion
      "description": "rotki usa Blockscout para obtener información sobre consultas de retiro de ETH y más. Solo es necesaria una clave API si las peticiones están siendo limitadas por el servidor de Blockscout.",
```

**Review by yabirgb — CHANGES_REQUESTED** [review]

**Review by kelsos — APPROVED** [review]

**Pull request merged**